includes/class-wc-stripe-feature-flags.php

```diff
@@ -7,10 +7,44 @@ class WC_Stripe_Feature_Flags {
 	const UPE_CHECKOUT_FEATURE_ATTRIBUTE_NAME = 'upe_checkout_experience_enabled';
 	const ECE_FEATURE_FLAG_NAME               = '_wcstripe_feature_ece';
 	const AMAZON_PAY_FEATURE_FLAG_NAME        = '_wcstripe_feature_amazon_pay';
+	const LPM_ACH_FEATURE_FLAG_NAME           = '_wcstripe_feature_lpm_ach';
+	const LPM_ACSS_FEATURE_FLAG_NAME          = '_wcstripe_feature_lpm_acss';
+	const LPM_BACS_FEATURE_FLAG_NAME          = '_wcstripe_feature_lpm_bacs';
 
-	const LPM_ACH_FEATURE_FLAG_NAME  = '_wcstripe_feature_lpm_ach';
-	const LPM_ACSS_FEATURE_FLAG_NAME = '_wcstripe_feature_lpm_acss';
-	const LPM_BACS_FEATURE_FLAG_NAME = '_wcstripe_feature_lpm_bacs';
+	/**
+	 * Map of feature flag option names => their default "yes"/"no" value.
+	 * This single source of truth makes it easier to maintain our dev tools.
+	 *
+	 * @var array
+	 */
+	protected static $feature_flags = [
+		'_wcstripe_feature_upe'            => 'yes',
+		self::ECE_FEATURE_FLAG_NAME        => 'yes',
+		self::AMAZON_PAY_FEATURE_FLAG_NAME => 'no',
+		self::LPM_ACH_FEATURE_FLAG_NAME    => 'no',
+		self::LPM_ACSS_FEATURE_FLAG_NAME   => 'no',
+		self::LPM_BACS_FEATURE_FLAG_NAME   => 'no',
+	];
+
+	/**
+	 * Retrieve all defined feature flags with their default values.
+	 *
+	 * @return array
+	 */
+	public static function get_all_feature_flags_with_defaults() {
```

Comment:
Is this function only being used currently for a dev tools scenario? Do you see any other use for this function going forward?

Reply:
Yes, it's only used in the context of the dev tools and I don't really see any other usage going forward 🤔. I added a note about it in [5999edb](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3930/commits/5999edbdb5589258a9ce19a2be276a568be5686d).

In the other PR I'm using the existence of this method to allow for loading the feature flags, but I guess I could also convert `$feature_flags` into `public static` and return all feature flags from `WC_Stripe_Feature_Flags::$feature_flags` directly?